apps/dashboard/test/system/files_test.rb

```diff
@@ -198,11 +204,11 @@ class FilesTest < ApplicationSystemTestCase
       find('tbody a', exact_text: File.basename(src_file), wait: MAX_WAIT)
       assert File.exist?(upload_file)
       assert_equal File.read(src_file), File.read(upload_file)
-      assert_equal File.stat(upload_file).mode, 33_188 # default 644
+      assert_equal(33_188, File.stat(upload_file).mode) # default 644
```

Comment:
The expected value is the first argument, so I had to switch these around just because the order's wrong.